objects/scienceoutpost/scienceinfobooth/scienceinfobooth.object

```diff
@@ -0,0 +1,91 @@
+    "Having trouble gaining research? Your passive research gain will increase the longer you play, and will also increase based on the tier of the planet you're on - Exploring more dangerous planets lets you gather research more quickly!",
+    "Okay, so a couple of coworkers were saying they were leaving to go snarsh some garflipps, and I asked them what that meant, and they looked at me like I was an idiot. Is that normal? Am I the wierd one for not knowing what that means?",
+	"A randomly-generated weapon will almost always be weaker than one you've crafted yourself. Crafted weapons also generally have special abilities designed to complement the weapon. Consider using a computer to turn randomly-generated weapons into research, instead!",
+    "Have a favorite weapon or armor set that you don't want to leave behind as you progress? Good news - you don't have to! Using your Tricorder's Upgrade function, you can use Upgrade modules to improve almost all Frackin' Universe weapons or armor. By doing this, you can ensure that any armor or weapon you like stays viable for the entire game!",
+    "Does an enemy seem almost invincible? It might just be resistant to the damage type you're using. Try switching things up with different weapons to find a weakness!",
+    "Don't know where to find a specific item or material? A Lab Directory can tell you if it's an output from any of your nearby machines. If that fails, check the official wiki displayed on the main menu!",
+    "Even carnivores have a reason to set up plant farms - many of the growable plants that Frackin' Universe adds can be eaten by anyone. Plus, Meatvine is a reliable source of nutrition, and many plants and their seeds can be extracted into useful materials!",
+    "Sorry we can't let you use most of the crafting and processing machines - We had to restrict access after Kevin tried to use them to create a gun that shot smaller versions of itself.",
+    "Most hazardous environmental effects can be prevented with either an immunity to that effect, or often through a high enough resistance to specific damage types.",
+    "Immunities and resistances are seperate - Even if you're immune to a planet's deadly heat, it won't reduce damage from fiery attacks unless you increase your heat resistance.",
+    "If things don't seem to be working right, first check to make sure you don't have any incompatible mods installed. If that doesn't work, post your full log file to the bug channel on the Frackin' Universe Discord.",
+    "Take your time - if you rush to advance as fast as possible, you'll miss all sorts of fun things - like these hints!",
+    "Mixing certain liquids can have unexpected results - do things right, and you can make more of almost any liquid!",
+	"Liquids will drain away if there's not a background block keeping them there - though oceans are an exception.",
```

Comment:
This is true of vanilla, but it took me an embarrassingly long time to realize it. With more space, I'd consider advising the player to use this to their advantage - either by adding background blocks to make raining liquids stick around for easy collection, or by removing background blocks to prevent flooding or drain pools of hazardous liquid.